### PR TITLE
gst-rtsp 1.4.5 (new formula)

### PR DESCRIPTION
**Pull request by cbenhagen**

### CODE REVIEW

File: Library/Formula/gst-rtsp.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  depends_on "gettext"
+  depends_on "gst-plugins-base"
+
+  def install
+    system "./configure",  "--disable-debug", "--disable-dependency-tracking",
+                           "--prefix=#{prefix}", "--disable-gtk-doc"
+    system "make"
+    system "make", "install"
+  end
+end
```
Comment (mikemcquaid) [inline Library/Formula/gst-rtsp.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/gst-rtsp.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class GstRtsp < Formula
+  homepage "http://gstreamer.freedesktop.org/"
+  url "http://gstreamer.freedesktop.org/src/gst-rtsp-server/gst-rtsp-server-1.4.5.tar.xz"
```
Comment (mikemcquaid) [inline Library/Formula/gst-rtsp.rb]: Can either the homepage or url be accessed over https?

Comment (cbenhagen) [inline Library/Formula/gst-rtsp.rb]: unfortunately not..

**Comment by DomT4** [comment]
Please add a test per Mike's comment, Thanks!

**Comment by DomT4** [comment]
This seems to have gone dead. Closing consequently.

If you pick up interest later please feel free to resubmit as a new PR with the comments above addressed, Thanks!

Thank you for the submission!

**Pull request closed without merging**